projects/josephus-problem/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Josephus Porblem in Every Language
+layout: default
+date: 2020-10-03
+last-modified: 2020-10-03
+featured-image: 
+tags: [recursion]
+authors:
+---

```

Comment:
Make sure to add yourself as an author! You'll need to put yourself in the authors.yml file as well.

Reply:
Done! Thank you